src/tests/ftest/io/ior_small.yaml

```diff
@@ -11,7 +11,7 @@ hosts:
     - client-F
     - client-G
     - client-H
-timeout: 1800
+timeout: 3000
```

Comment:
How about reducing the block size for the case of transfer size '256B' from '4M' to '2M' if that sounds reasonable?
It is 'ior -POSIX -t 256B -b 4M' which seem to be taking around 20 mints on it's own. May be then hopefully it will take 10 mints.
Increasing timeout from already 30mints to 50 mints will only result in longer queues over the time.

Reply:
I'm ok with that. @ashleypittman ?

Reply:
Actually, IOR has a -T option we could use, "maxTimeDuration – max time in minutes to run tests".  Given this is ior_"small" then adding -T 5 on the command line would seem reasonable.

Reply:
well we want a complete cycle of writing, reading and verifying the data.
we don't want to cut off the run time in the middle of the run.
we could try stonewalling option to write, but that requires adding more options and a stonewalling file, and im just looking for a quick fix for this.

Reply:
Agreed, I don't know how -T interacts with the data integrity tests.  It would probably be a better solution, but time-to-solution is more important given this is currently causing failures.